packages/lending/src/adapter.ts

```diff
@@ -159,11 +159,17 @@ export class Adapter extends common.Web3Toolkit {
    * Retrieves an array of portfolios for a given account from all registered protocols.
    *
    * @param {string} account - The account identifier for which portfolios are to be retrieved.
+   * @param {Object} [options] - Optional parameters.
+   * @param {string|number} [options.blockTag] - An optional block tag
    * @returns {Promise<Portfolio[]>} A promise that resolves to an array of Portfolio objects from all protocols.
    */
-  async getPortfolios(account: string): Promise<Portfolio[]> {
+  async getPortfolios(account: string, options?: { blockTag?: string | number }): Promise<Portfolio[]> {
     const portfolios = await Promise.all(
-      Object.values(this.protocolMap).map((protocol) => protocol.getPortfolios(account))
+      Object.values(this.protocolMap).map((protocol) => {
+        if (options?.blockTag) protocol.setBlockTag(options.blockTag);
```

Comment:
Should we always set the block tag? If I want to retrieve the current block but have previously set a block tag, the system will use the prior block tag setting instead of fetching the latest block.

Reply:
Good point. It seems that when the client doesn't pass the block tag, we need to clear the block tag so that it defaults to the latest block.